src/main/finder.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2015-2016 Yuya Ochiai
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export default class Finder {
```

Comment:
Just curious, is it possible to can implement this as a React component? Almost of other components are made as React components, so it would be consistent.

Reply:
Yes, it can be implemented in react as well 👍